basic-sql-operations.md

```diff
@@ -100,7 +100,7 @@ For example, to create a table named `person` which includes fields such as numb
 
 ```sql
 CREATE TABLE person (
-    id INT(11),
+    id int,
```

Comment:
```suggestion
    id INT,
```

Ensure consistent capitalization of SQL data types (`INT`, `VARCHAR `, `DATE`) in this SQL statement.

Reply:
@dveeden PTAL